internal/resolvers/youtube/resolver.go

```diff
@@ -51,6 +64,26 @@ func New(cfg config.APIConfig) (resolvers []resolver.CustomURLManager) {
 		return
 	}
 
+	resolvers = append(resolvers, resolver.CustomURLManager{
```

Comment:
Add a comment showing what sort of links we handle here

Reply:
Commented.